src/ActionModal/components/UndelegationModal.vue

```diff
@@ -450,7 +449,35 @@ export default {
         return !this.address
       },
     },
-
+    totalStaked: {
+      query: gql`
+        query overview($networkId: String!, $address: String!) {
```

Comment:
please don't use the overview query. I would like to remove that query soon.

Reply:
Thanks for replacing